streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -722,22 +721,20 @@ private void checkIfUnexpectedUserSpecifiedConsumerConfig(final Map<String, Obje
      * except in the case of {@link ConsumerConfig#BOOTSTRAP_SERVERS_CONFIG} where we always use the non-prefixed
      * version as we only support reading/writing from/to the same Kafka Cluster.
      *
-     * @param streamThread the {@link StreamThread} creating a consumer
      * @param groupId      consumer groupId
      * @param clientId     clientId
      * @return Map of the consumer configuration.
      */
-    public Map<String, Object> getConsumerConfigs(final StreamThread streamThread,
-                                                  final String groupId,
+    public Map<String, Object> getConsumerConfigs(final String groupId,
```

Comment:
I felt it is better to explicitly add it in `StreamThread` because 1) we do not need to include the dependency of `TaskManager` in `StreamsConfig`, and 2) In all unit tests of `StreamsConfig` we call this function with `null` as in this parameter, makes me think that it is not necessarily a passed-in parameter.

Reply:
Ack.